package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "my-calculator",
+  "version": "1.0.0",
+  "description": "A basic client side calculator",
+  "main": "calculator.html",
+  "scripts": {
+    "test": "node node_modules/mocha/bin/mocha spec --recursive"
```

Comment:
We can run `npm test` from the command line and it will execute this command. This runs a node instance which runs mocha. It picks up all the files in the spec folder (`spec`), and also recursively searches sub folders (`--recursive`).